src/hooks/use-site-details.tsx

```diff
@@ -285,14 +285,27 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 			try {
 				updatedSite = await getIpcApi().startServer( id );
 			} catch ( error ) {
-				getIpcApi().showErrorMessageBox( {
-					title: __( 'Failed to start the site server' ),
-					message: __(
-						"Please verify your site's local path directory contains the standard WordPress installation files and try again. If this problem persists, please contact support."
-					),
-					error,
-					showOpenLogs: true,
-				} );
+				if (
+					error instanceof Error &&
+					error.message.includes( 'Studio failed to start the proxy server' )
+				) {
+					getIpcApi().showErrorMessageBox( {
+						title: __( 'Studio failed to initialize custom domains' ),
+						message: __(
+							'Studio needs to use port 80, but it’s already in use by another app. Close any local development apps and restart Studio.'
```

Comment:
Should we mention that the port 80 is only necessary for custom domains?

Reply:
The title says it failed to initialize custom domains, but it makes sense to clarify the message. I've added that in https://github.com/Automattic/studio/pull/1089/commits/c82c9028335521850b766e5467dcf6620877fe08